src/rabbitstressor.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
So would it be better to rename the file as `stressor.js` or `queuestressor.js`?
Also, would it still make sense to keep the stressor tests under `test/`?